src/charm.py

```diff
@@ -396,37 +397,6 @@ def _join_unit_to_cluster(self) -> None:
             self.unit.status = WaitingStatus("waiting to join the cluster")
             logger.debug("waiting: failed to acquire lock when adding instance to cluster")
 
-    def _remove_scaled_down_units(self) -> None:
-        """Remove scaled down units from the cluster."""
-        current_units = 1 + len(self.peers.units)
-        cluster_status = self._mysql.get_cluster_status()
-        if not cluster_status:
-            self.unit.status = BlockedStatus("Failed to get cluster status")
-            return
-
-        try:
-            addresses_of_units_to_remove = [
-                member["address"]
-                for unit_name, member in cluster_status["defaultreplicaset"]["topology"].items()
-                if int(unit_name.split("-")[-1]) >= current_units
-            ]
-        except ValueError:
-            # exception can occur if unit is not yet labeled
-            return
-
-        if not addresses_of_units_to_remove:
-            return
-
-        self.unit.status = MaintenanceStatus("Removing scaled down units from cluster")
-
-        for unit_address in addresses_of_units_to_remove:
-            try:
-                self._mysql.force_remove_unit_from_cluster(unit_address)
```

Comment:
we should delete the `force_remove_unit_from_cluster` method as well

Reply:
c28ae64